az-digital/az_quickstart
# Allow content editors to edit any media

## Description
This PR adds edit any media permissions for all media types. We recently noticed that content editors could only edit their own media even though they can edit all content types.

Not certain if this update will require a database update?

## Related Issue
#1014

## How Has This Been Tested?
Locally and in Probo

To test:
- Log in as azadmin and create content with media (text on media, photo gallery, any content type)
- Log out and log in as azcontenteditor. Verify that you still see the edit icon on the associated media and can edit it.

## Types of changes


## Checklist:
